docs/source/configuration/apollo-telemetry.mdx

```diff
@@ -304,6 +304,16 @@ You can configure whether the Apollo Router reports GraphQL error information to
 - To prevent your router from reporting error information at all, you can set the `send` option to `false`.
 - To include all error details in your router's reports to GraphOS, you can set the `redact` option to `false`.
 
+<blockquote>
+
+**Your subgraph libraries must support federated tracing (also known as FTV1 tracing) to provide errors to GraphOS.**
+
+- To confirm support, check the `FEDERATED TRACING` entry for your library on [this page](/federation/building-supergraphs/supported-subgraphs).
+- Consult your library's documentation to learn how to enable federated tracing.
```

Comment:
```suggestion
- If is federated tracing isn't yet enabled, consult your library's documentation to learn how to enable it.
```

Is there a link or generic instruction we can point readers to for "your library's documentation"? It could be that this is obvious to readers already and we don't need it. (Still learning the ropes 😅.)

Reply:
Not really since "your library" would be what every GraphQL library they are using